app/controllers/register/beneficiaries/charityortrust/charity/AddressInTheUkYesNoController.scala

```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```

Comment:
should be import controllers.actions.register.**charity**.NameRequiredAction

Reply:
Updated